app/controllers/posts_controller.rb

```diff
@@ -4,6 +4,20 @@ def index
     @posts = @user.posts
   end
 
+  def new
+    @post = Post.new
+  end
+
+  def create
+    @post = Post.new(author: current_user, title: params[:post][:title], text: params[:post][:text])
```

Comment:
- [ ] [OPTIONAL] Good job implementing this, but please be aware that you can create a private method that can be reused. You can read more about it below

`https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides`